{{cookiecutter.project_name}}/project.yml

```diff
@@ -7,9 +7,21 @@ copyright: {{cookiecutter.copyright}}
 modulepath: libs
 downloadpath: libs
 repo:
-    - type: package
-      url: {{cookiecutter.package_repo_url}}
     - type: git
       url: {{cookiecutter.repo_url}}
+    # Index url for V2 modules
+    # This is for <=ISO6
+    - type: package
+      url: {{cookiecutter.package_repo_url}} 
+pip: # for >=ISO7 
+  # Index url for python package and V2 modules
+  index_url: {{cookiecutter.package_repo_url}}
+  # Extra index urls for python package and V2 modules
+  extra_index_url: []
+  # Allow pip to install prerelease versions or not?
+  pre: {{cookiecutter.install_mode != "release"}}
```

Comment:
I don't love it but here I think we may not have a choice if we want to keep it simple.

Reply:
I agree